src/AvatarMember.jsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import cx from 'classnames';
 import Avatar from './Avatar';
+import { getIconAsBase64Uri } from './utils/base64';
+
+const NO_PHOTO_SRC = getIconAsBase64Uri('profile');
```

Comment:
We're rendering the icons from an SVG sprite, so there isn't a static SVG asset to point to in the `background-image` rule